Skeletorfile.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+use NiftyCo\Skeletor\Skeletor;
+
+return function (Skeletor $skeletor) {
+    $skeletor->intro('Starting Fission setup...');
+
+
+    if (!$skeletor->exists('.env')) {
+        $env = $skeletor->readFile('.env.example');
+        $skeletor->writeFile('.env', $env);
+        $skeletor->info('✓ .env file created.');
+    }
+
+    $skeletor->spin('Generating application key...', function () use ($skeletor) {
+        $skeletor->exec(['php', 'artisan', 'key:generate', '--ansi']);
+    });
+    $skeletor->info('✓ Application key generated.');
+
+    if (!$skeletor->exists('node_modules')) {
+        $skeletor->spin('Running npm install...', function () use ($skeletor) {
+            $skeletor->exec(['npm', 'install']);
+        });
+        $skeletor->info('✓ npm dependencies installed.');
+    }
+
+    if ($skeletor->exists($authPath = $_SERVER['HOME'] . '/Code/flux-auth.json')) {
```

Comment:
I found this to be a bit wonky while doing this because I don't store a copy of my flux auth here so this code path may never execute for most, so we could just skip it for the branch of asking for email/key.